src/main/java/homework_4/custom_file_reader/Main.java

```diff
@@ -0,0 +1,9 @@
+package homework_4.custom_file_reader;
+
+public class Main {
+    public static void main(String[] args) {
+        new CustomFileReader().run1();
+        new CustomFileReader().run2();
+        new CustomFileReader().run3();
+    }
```

Comment:
Is it expected? 
![image](https://user-images.githubusercontent.com/58872509/134551330-2add47eb-3ab3-40ca-a502-d76dbc162706.png)